.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Husky?

Reply:
It's a git hook package. Afaik it's the best way for running pretty-quick in pre-commit.
Also it has zero dependencies, so it shouldn't be too bad to add as a dev dependency.